### PR TITLE
Add FoundIn labels for Chrome bugs.

### PR DESCRIPTION
**Pull request by adetaylor**
The `FoundIn=NN` label is a standard for marking which milestone
in which functional bugs are found within Monorail.
ClusterFuzz should adopt this standard for the following reasons:

1) Why not? ClusterFuzz has this information so it would be great
   to add it.
2) Sheriffbot currently relies on `Security_Impact` when deciding on
   merge decisions. Currently, `Security_Impact`s are 'promoted'
   when a new build goes to stable or to beta. This sometimes is
   problematic when a single milestone is both stable and beta.
3) More generally, switching the security sheriffing procedures
   from Security_Impact to FoundIn would be less confusing.

A future change would ideally also set RegressedIn, but that's
a more significant project.

For more information internally within Google, see
https://goto.google.com/clusterfuzz-foundin

### CODE REVIEW

File: src/appengine/requirements.txt [hunk 1]
```diff
@@ -6,3 +6,6 @@ PyJWT==1.7.1
 pyOpenSSL==19.1.0
 requests-toolbelt==0.9.1
 sendgrid==6.0.4
+mock==4.0.2
```
Comment (inferno-chromium) [inline src/appengine/requirements.txt:9]: These are not required, already in Pipfile in clusterfuzz root.

File: src/appengine/libs/issue_management/issue_filer.py [hunk 2]
```diff
@@ -178,14 +178,30 @@ def update_issue_impact_labels(testcase, issue):
...
 
   if existing_impact != data_types.SecurityImpact.MISSING:
     issue.labels.remove('Security_Impact-' + impact_to_string(existing_impact))
 
   issue.labels.add('Security_Impact-' + impact_to_string(new_impact))
+  update_issue_foundin_labels(testcase, issue, milestones_foundin)
+
+
+def update_issue_foundin_labels(testcase, issue, milestones_foundin):
+  labels = [label.lower() for label in issue.labels]
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/issue_filer.py:200]: Add function description.

File: src/appengine/libs/issue_management/issue_filer.py [hunk 3]
```diff
@@ -178,14 +178,30 @@ def update_issue_impact_labels(testcase, issue):
     # No impact information.
     return
 
+  versions_foundin = [
```
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/issue_filer.py:181]: I think this calculation can just move inside update_issue_foundin_labels, so milestones_foundin arg can be removed.

File: src/appengine/libs/issue_management/issue_filer.py (hunk 2 above)
Comment (inferno-chromium) [inline src/appengine/libs/issue_management/issue_filer.py:200]: Can you add a bailout on:
if not testcase.is_impact_set_flag:
  return

**Review by inferno-chromium — APPROVED** [review]
LGTM with some minor changes.

File: src/appengine/libs/issue_management/issue_filer.py [hunk 4]
```diff
@@ -178,14 +178,30 @@ def update_issue_impact_labels(testcase, issue):
...
     issue.labels.remove('Security_Impact-' + impact_to_string(existing_impact))
 
   issue.labels.add('Security_Impact-' + impact_to_string(new_impact))
+  update_issue_foundin_labels(testcase, issue, milestones_foundin)
+
+
+def update_issue_foundin_labels(testcase, issue, milestones_foundin):
+  labels = [label.lower() for label in issue.labels]
+  for found_milestone in milestones_foundin:
+    if 'foundin-%s' % found_milestone in labels:
```
Comment (oliverchang) [inline src/appengine/libs/issue_management/issue_filer.py:202]: nit: let's start using Python 3 f-strings. i.e. `f'foundin-{found_milestone}'`

File: src/appengine/requirements.txt (hunk 1 above)
Comment (oliverchang) [reply src/appengine/requirements.txt:9]: +1. Were you running into issues otherwise?

Comment (adetaylor) [reply src/appengine/requirements.txt:9]: I was, but I'll revert this commit and e-mail about them instead.

**Review by inferno-chromium — APPROVED** [review]

**Pull request merged**